### ISSUE TITLE
Numpy incompatibility, it's a 2 second fix

### ISSUE THREAD

**Issue by mojg** [scikit-optimize/scikit-optimize#1147]
There are numpy incompatibility in the addresses below:
https://github.com/scikit-optimize/scikit-optimize/blob/a2369ddbc332d16d8ff173b12404b03fea472492/skopt/space/transformers.py#L262
https://github.com/scikit-optimize/scikit-optimize/blob/a2369ddbc332d16d8ff173b12404b03fea472492/skopt/space/transformers.py#L275

**Comment by ChromaticIsobar** [comment]
Yes, it would be a very simple fix. You may want to refer to issue #1138 for further discussion.
This may be a duplicate.

**Comment by RNarayan73** [comment]
A workaround that worked for me is to put these lines before fitting the estimator:

import numpy as np
np.int = int

**Comment by ChromaticIsobar** [comment]
The workaround is mentioned in issue #1138 
Anyway, this project may be dead, since last commit on master was three years ago
